.github/workflows/deploy_to_s3_prod.yml

```diff
@@ -36,15 +36,17 @@ jobs:
         NODE_ENV: Production
         VUE_APP_AF_API_KEY: ${{ secrets.VUE_APP_AF_API_KEY }}
         VUE_APP_BASE_URL_PLIO: ${{ secrets.VUE_APP_BASE_URL_PLIO }}
+        VUE_APP_AWS_ACCESS_KEY_ID: ${{ secrets.VUE_APP_AWS_SQS_ACCESS_KEY }}
+        VUE_APP_AWS_SECRET_KEY: ${{ secrets.VUE_APP_AWS_SQS_SECRET_ACCESS_KEY }}
```

Comment:
How is this working?
In your .env file and in your JS code, you'll be accessing these variables as `process.env.VUE_APP_AWS_SQS_ACCESS_KEY` and `process.env.VUE_APP_AWS_SQS_SECRET_ACCESS_KEY` because that is what's defined in your .env file.

But this github workflow file doesn't expose any variables with these names, it instead exposes these two variables `VUE_APP_AWS_ACCESS_KEY_ID, VUE_APP_AWS_SECRET_KEY`

Reply:
So, I added two secrets to environment variables, which are `VUE_APP_AWS_SQS_ACCESS_KEY` and `VUE_APP_AWS_SQS_SECRET_ACCESS_KEY `. These are the secrets the code uses. You can check it in Settings > Environment. The repo secrets are `VUE_APP_AWS_ACCESS_KEY_ID, VUE_APP_AWS_SECRET_KEY`. The code does not use this. The workflow only does for deploying to S3.

Reply:
So, I added two secrets to environment variables, which are `VUE_APP_AWS_SQS_ACCESS_KEY` and `VUE_APP_AWS_SQS_SECRET_ACCESS_KEY` . These are the secrets the code uses. You can check it in Settings > Environment. The repo secrets are `VUE_APP_AWS_ACCESS_KEY_ID, VUE_APP_AWS_SECRET_KEY`. The code does not use these secrets. It's to deploy to S3 by the actions.

Reply:
<img width="473" alt="Screenshot 2021-09-15 at 12 12 28 PM" src="https://user-images.githubusercontent.com/20169829/133383602-103e839d-a708-48c7-a9c0-52840f1117d7.png">

no no I think there's some confusion. if you see the screenshot above, you'll see that to access the SQS, the code uses the variables with these names
```
VUE_APP_AWS_SQS_ACCESS_KEY
VUE_APP_AWS_SQS_SECRET_ACCESS_KEY
```

But how is the deployed app finding these variables? In the screenshot below, you can see that while building the app, the env variables are being correctly picked up from github secrets but the name of those variables is 
```
VUE_APP_AWS_ACCESS_KEY_ID
VUE_APP_AWS_SECRET_KEY
```
<img width="629" alt="Screenshot 2021-09-15 at 12 14 20 PM" src="https://user-images.githubusercontent.com/20169829/133383845-fbdc8696-9dfb-419c-9273-29c432238057.png">

The JS code doesn't use the S3 access keys right? It just uses the keys to access SQS. 
I guess we can discuss this in the standup